Slim/Player/Song.pm

```diff
@@ -445,19 +446,9 @@ sub open {
 		};
 	}
 	
-	# now can set seekdata support information
-	$seekdata->{'sourceStreamLength'} = $track->audio_size;
-	
-	if ($transcoder->{'stripHeader'}) {
-		# Set initialAudioBlock to 0 (not undef) so that File does not re-send it
-		$self->initialAudioBlock(0);
-		# Tell File and HTTP to strip header from source
-		$seekdata->{'sourceHeaderOffset'} = $track->audio_offset || 0,
-		main::INFOLOG && $log->info( "stripping header of ", $seekdata->{'sourceStreamOffset'}, " / ", $seekdata->{'sourceStreamLength'});
-	}
-	
-	# update seekdata (might be only support information)
-	$self->seekdata($seekdata);
+	# don't modify $song in getConverterCommand2 
+	$self->stripHeader($transcoder->{'stripHeader'});
+print("TRANSCODING using $transcoder->{'command'} FROM $format to $transcoder->{'streamformat'} strip:", $self->stripHeader, " ofs:", $track->audio_offset, "\n");		
```

Comment:
Should this "print" be converted to logger call.

Reply:
It should not have made its way to the PR, I'm using these prints for internal debugging